src/PowerShellEditorServices/Services/PowerShell/Execution/SynchronousPowerShellTask.cs

```diff
@@ -16,7 +16,14 @@
 
 namespace Microsoft.PowerShell.EditorServices.Services.PowerShell.Execution
 {
-    internal class SynchronousPowerShellTask<TResult> : SynchronousTask<IReadOnlyList<TResult>>
+    internal interface ISynchronousPowerShellTask
+    {
+        PowerShellExecutionOptions PowerShellExecutionOptions { get; }
+
+        void MaybeAddToHistory();
```

Comment:
Added this because now we can use the interface in a place we were already trying to test for it.